src/objects/tests/test_object_api.py

```diff
@@ -15,14 +15,16 @@
 from objects.utils.test import TokenAuthMixin
 
 from .constants import GEO_WRITE_KWARGS
-from .utils import mock_objecttype
+from .utils import mock_objecttype, mock_objecttype_version
 
-OBJECT_TYPE = "https://example.com/objecttypes/v1/types/a6c109"
+OBJECT_TYPE = "https://example.com/objecttypes/v1/types/a6c109/versions/1"
```

Comment:
I think this url should be without `/versions/1`

Reply:
correct!